### ISSUE TITLE
Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type...

### ISSUE THREAD

**Issue by bgibb95** [onfido/onfido-sdk-ui#2140]
### What was the expected behaviour?

A successful build of an Angular application which uses `onfido-sdk-ui` as a dependency was expected.

### What happened instead?

An error was emitted after running `ng build` for our Angular application. The error is:

```
Error: node_modules/@types/node/ts4.8/globals.d.ts:72:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; }'.

72 declare var AbortSignal: {
               ~~~~~~~~~~~

  node_modules/@typescript/lib-dom/index.d.ts:2074:13
    2074 declare var AbortSignal: {
                     ~~~~~~~~~~~
    'AbortSignal' was also declared here.

Process finished with exit code 1
```

This occurs when trying to update `onfido-sdk-ui` to `v10.0.0` or above.

In our case, we are trying to update to `v10.3.1` from `v8.0.0`.

Steps to try to resolve the issue included:
1. Removing all node modules and reinstalling with `npm i`.
2. Updating `@types/node` to the latest version and then running step 1 again.

None of which were successful.

### Version info:

- Onfido SDK - `v10.3.1`
- OS - `masOS Ventura - v13.1 (22C65)`
- Browser - `Chome Version 109.0.5414.87 (Official Build) (arm64)`
- Device - `MacBook Pro Apple M1 Pro 2021`
- Node - `v16.18.0`
- Framework or relevant dependencies
  - `"typescript": "~4.8.4"`
  -  `"@types/node": "^18.11.18"`
  -  `"@angular/cdk": "^14.2.7"`
  - `"@angular/common": "^14.2.12"`
  - `"@angular/compiler": "^14.2.12"`
  -  `"@angular/core": "^14.2.12"`
  -  `"@angular/cli": "^14.2.10"`
  -  `"@angular/compiler-cli": "^14.2.12"`
  - `"@angular/language-service": "^14.2.12"`
  - `"onfido-sdk-ui": "10.3.1"`

### Integration configuration:

N/A

### Steps to reproduce:

1. Run `npm i` with an Angular 14 project which uses Onfido SDK UI `v10.3.1` as a dependency.
2. Run `ng build` for the Angular application.
4. See the error in the terminal.

### Reproducible demo:

![Screenshot 2023-01-18 at 4 38 09 PM](https://user-images.githubusercontent.com/25341501/213199957-6e5b5fee-87c1-423f-9934-c91da0933567.png)

![Screenshot 2023-01-18 at 4 42 27 PM](https://user-images.githubusercontent.com/25341501/213200918-3b123444-8b4d-4de8-82c1-62254829ce98.png)

**Comment by warrenseine** [comment]
You are depending on `@types/node`. You should not if you build a web application. Remove it and the conflict will disappear.

**Comment by bgibb95** [comment]
Thanks for the response @warrenseine. Removing `@types/node` from the `package.json` file did not resolve the issue.

The issue has been resolved though for our application.

We found that `"node"` was added in the `types` array for the `compilerOptions` in our Angular application's `tsconfig.json` file.

Removing `"node"` from the `types` array results in the build succeeding.

I think we can close this issue for now.
